packages/playwright/src/common/config.ts

```diff
@@ -90,6 +90,7 @@ export class FullConfigInternal {
     this.config = {
       configFile: resolvedConfigFile,
       rootDir: pathResolve(configDir, userConfig.testDir) || configDir,
+      failOnFlakyTests: takeFirst(configCLIOverrides.failOnFlakyTests, userConfig.failOnFlakyTests, false),
```

Comment:
Let's remove `FullConfigInternal#cliFailOnFlakyTests` in favour of this.

Reply:
Just to be clear, do you want me to remove the CLI option? Or should I keep things as is, the CLI flag takes priority over the config file?

Reply:
I want you to remove the internal representation of the CLI option, not the option itself. What you added looks good.